server/pkg/schema/field_tag.go

```diff
@@ -16,11 +16,11 @@ func NewFieldTag() *FieldTag {
 	}
 }
 
-func FieldTagFrom(values *[]string, defaultValue *int) (*FieldTag, error) {
+func FieldTagFrom(values []string, defaultValue *int) (*FieldTag, error) {
```

Comment:
For tag property, defaultValue should be string, but it is integer... Other type property may have the same mistakes.

Reply:
It's an integer because it holds the index of the default value.
```
values = ["a", "b", "c"]
defaultValue = 0  // this means the actual default value is "a"
```

Reply:
I think value should be string. Index is not good because if values changes, that index no longer makes sense.